### PR TITLE
add color passing to embedService

### PR DESCRIPTION
**Pull request by dcepil**
updated `embedService.js`:
- now accepts `color` as an option, making it possible to change the previously static embed color

### CODE REVIEW

**Comment by dcepil** [comment]
[Implement passing a color to embedService.js](https://trello.com/c/YcRY2RXc/29-implement-passing-a-color-to-embedservicejs)

File: services/embedService.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ const argsTitle = (args) => {
     : argsArray;
 };
 
-const embedTemplate = (msg) => {
+const embedTemplate = (msg, color = 16750462) => {
   return {
-    color: 16750462,
+    color,
```
Comment (VuongRon) [inline services/embedService.js:11]: I don't think this is necessary, later when `Object.assign(embedTemplate(msg), options)` is done, the duplicate keys from the 2nd arg (options) will override the first I believe.

**Review by VuongRon — COMMENTED** [review]
Investigate 🕵️

File: services/embedService.js (hunk 1 above)
Comment (dcepil) [reply services/embedService.js:11]: True.

**Pull request closed without merging**